tcms/testruns/tests/test_admin.py

```diff
@@ -18,76 +19,81 @@ def setUpTestData(cls):
 
         cls.superuser = UserFactory()
         cls.superuser.is_superuser = True
-        cls.superuser.set_password("password")
+        cls.superuser.set_password('password')
```

Comment:
This is unrelated to the issue at hand and causes black to fail. Rebase to master and always use `black tcms/` to format your source code before pushing.